modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -67,9 +67,8 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::validParams()
 
   MooseEnum extrapolated_lower_limit_behavior(
       "ERROR WARN IGNORE EXCEPTION DONOTHING USELIMIT EXTRAPOLATE", "EXTRAPOLATE");
-  // Forbid extrapolation of on high end of limit
-  MooseEnum extrapolated_upper_limit_behavior("ERROR WARN IGNORE EXCEPTION DONOTHING USELIMIT",
-                                              "USELIMIT");
+  // Only allow ERROR and EXCEPTION on upper bounds
+  MooseEnum extrapolated_upper_limit_behavior("ERROR EXCEPTION", "EXCEPTION");
```

Comment:
You'll have to change make a new copy of `error_limit_behavior` for `cell_input_window_high_failure`, `wall_input_window_high_failure`, and `old_strain_input_window_high_failure` too actually...

Reply:
good catch!